Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -207,6 +207,13 @@
           "SubId": "Odbc",
           "ContentsRegex": "$MySqlConnectionStringAdo",
           "MessageArguments": { "secretKind": "ADO or ODBC MySQL connection string" }
+        },
+        {
+          "Id": "SEC101/045",
+          "Name": "DoNotExposePlaintextSecrets/PostgreSqlConnectionString",
+          "SubId": "Ado",
+          "ContentsRegex": "$PostgreSqlConnectionStringAdo",
+          "MessageArguments": { "secretKind": "ADO PostgreSQL connection string" }
         }
```

Comment:
remove this, since a few lines above we have the same

Reply:
Done.  This removed a lot of redundancies in the respective sarif file too.

---
In reply to: [569782450](https://github.com/microsoft/sarif-pattern-matcher/pull/159#discussion_r569782450) [](ancestors = 569782450)